controller/schema/errors.go

```diff
@@ -27,7 +27,7 @@ type ValidationErrors struct {
 }
 
 func (e ValidationErrors) Error() string {
-	return fmt.Sprintf("schema validation failed")
+	return "schema validation failed"
```

Comment:
nothing of use in the `ValidationErrors` to return back to "whomever" gets this error? Pls confirm that this is "all the useful information"? :)

Reply:
Change was for the linter once again - said that the `Sprintf` wasn't needed. 

This specifically is a JSON marshaled object - so the logic to output useful information is in the `MarshalJSON` function which is a few lines above.